pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -175,20 +170,18 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 
 	// get drive serial number
 	sn := drive.SerialNumber
-	vid := drive.VID
-	pid := drive.PID
-	for _, l := range lsblkOut {
-		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(l.Vendor, vid) &&
-			strings.EqualFold(l.Model, pid) {
-			device = l.Name
-			break
+	for _, blockDevice := range lsblkOut {
+		if strings.EqualFold(blockDevice.Serial, sn) {
+			return blockDevice.Name, nil
 		}
 	}
 
-	if device == "" {
-		errMsg := fmt.Errorf("unable to find drive path by SN %s, VID %s, PID %s", sn, vid, pid)
-		return "", errMsg
+	// device path might be already set by hwmgr
+	if drive.Path != "" {
+		l.log.Warnf("unable to find a drive path by a given SN: %s, using a drive path: %s", sn, drive.Path)
+		return drive.Path, nil
 	}
 
-	return device, nil
+	errMsg := fmt.Errorf("unable to find drive path by a SN %s and a drive path", sn)
```

Comment:
maybe that slogan is better? "unable to find drive path by a SN %s or in DriveCR?"